### PR TITLE
add keyboard support intro screens

### LINKED ISSUE #1936: Add keyboard support for intro screens
**Is your feature request related to a problem? Please describe.**
It's very annoying when you're on desktop and can't get through the intro screens quickly because you have to reach for your mouse (yes, I have a split keyboard so I'd need three hands...).

**Describe the solution you'd like**
- Allow moving language selection with Up and Down arrow keys
- Allow selecting language with Enter
- Allow moving on from the explanation screen with Enter - so same effect as the action button 'Continue'

**Describe alternatives you've considered**
"Only focus on mobile" is what we said.  
The exception I'm proposing here is for debugging purposes / devex.  
I find debugging on desktop a lot more convenient. There is no ADB to setup, and you can copy/paste to other tools and issue tracker. Going through the onboarding screens is a small annoyance that builds up over time.

**Additional context**

image

image

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1936

### CODE REVIEW

**Review by tijlleenders — APPROVED** [review]
Yeah, this works!

@Tushar-4781 Code OK?

File: src/hooks/useKeyPress.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState, useCallback } from "react";
+
+export const useKeyPress = (targetKey: string) => {
+  const [keyPressed, setKeyPressed] = useState(false);
+
+  const downHandler = useCallback(
```
Comment (Tushar-4781) [inline src/hooks/useKeyPress.ts:6]: you can create a single handler and perform this  `setKeyPressed(event.type === "keydown");`

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/hooks/useKeyPress.ts (hunk 1 above)
Comment (vinaybadgujar102) [reply src/hooks/useKeyPress.ts]: @Tushar-4781, updated the code based on your suggestion.

**Comment by tijlleenders** [comment]
@Tushar-4781 *bump*

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**